schema/omega-services.async.api.yaml

```diff
@@ -48,6 +48,17 @@ channels:
         oneOf:
           - $ref: '#/components/messages/getEditSet'
 
+  omega.editorial.web.application.instance.1:
+    description: The queue on which response messages will be sent.
+    publish:
+      operationId: omega_editorial_web_application_instance_1
+      description: Response messages
+      message:
+        oneOf:
+          - $ref: '#/components/messages/editSet'
+          - $ref: '#/components/messages/editSetSummary'
+          - $ref: '#/components/messages/editSetRecord'
```

Comment:
I think we want `updateEditSetRecord` here as well. Basically anything that can appear in this queue.

Reply:
I've added the update message here, as well as the update message confirmation